website/source/docs/configuration/interpolation.html.md

```diff
@@ -61,6 +61,13 @@ attribute set, you can access individual attributes with a zero-based
 index, such as `${aws_instance.web.0.id}`. You can also use the splat
 syntax to get a list of all the attributes: `${aws_instance.web.*.id}`.
 
+#### Outputs from a data source
```

Comment:
It may sound like a nitpick, but to avoid confusions in the naming conventions, would you mind changing this to `Attributes of a data source`? We use "outputs" in slightly different context below.